dealers_choice.html

```diff
@@ -0,0 +1,117 @@
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3023.5875876925948!2d-73.98521994863147!3d40.72709447922883!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c259a9610bfc8f%3A0x58666551a9dcb450!2sRosella!5e0!3m2!1sen!2sus!4v1650764749363!5m2!1sen!2sus" width="600" height="450" style="border:black solid 2px;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>       
+            <p>This East Village restaurant almost exclusively uses ingredients found within the United States. They dole out bigeye tuna from North Carolina, striper raised in Bushwick, and incredible fluke sashimi that was caught so close to Rosella it could have taken the afternoon train from Montauk while enjoying a tallboy in a brown paper bag. If you’ve been carrying around the misconception that American fish is only for guys on Hinge to pose with in their profile pictures, come to this luxurious-but-lowkey restaurant, eat some Washington State Arctic Char over a bed of California-grown rice, and be proven wrong. An à la carte meal at Rosella will be the highlight of your week (and run you around $50), but the $150 omakase is particularly impressive.</p>
+            
+            </div>
+        </div>
+
+        <div class="post">
+            <div>
+            <h2>3. Shion 69 Leonard Street</h2>
+            <a target="_blank" href="https://www.69leonardstreet.com/">RESTAURANT WEBSITE</a>
+            <br>
+            <img src="https://infatuation.imgix.net/Steamed%20Managatsuo%20(c.%20Vincent%20Zhu).jpg?ar=4%3A3&auto=format&crop=entropy&fit=crop&q=65&w=640&ixlib=react-9.3.0" alt="sushi in bowl"/>
+            <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3024.018920585852!2d-74.00759404863173!3d40.717599979229895!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c25a203e04b19d%3A0x71da706a09467037!2sShion%2069%20Leonard%20Street!5e0!3m2!1sen!2sus!4v1650765346966!5m2!1sen!2sus" width="600" height="450" style="border:black solid 2px" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+            <p>At $420 a pop, Shion 69 Leonard Street is one of the most expensive omakases in the city. The price tag alone makes it intriguing for a special occasion—but throwing back sake and singing "Happy Birthday" here would be akin to doing TikTok dances in a museum. This place is for one of those mature milestones, the kind that are straight out of a Neil Young song, when you want to sit and reflect on life, make peace with past failures and ones who got away. And you'll get about 15 minutes to do that, before it's time to pay attention. Between the peaceful room, precise salt-curing, and rhythmic pace of hot and cold plates, this meal will demand your presence like it's the cult leader of a silent meditation retreat.</p>
+            </div>
+        </div>
+
+        <div class="post">
+            <div>
```

Comment:
Try using better formatting, (indentation) here for best code practice and readability.